Sources/Responses/Classification.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Classification.swift
+//  BoxSDK-iOS
+//
+//  Created by Sujay Garlanka on 5/1/20.
+//  Copyright © 2020 box. All rights reserved.
+//
+
+import Foundation
+
+/// Details about the classification applied to a Box file or folder
+public class Classification: BoxInnerModel {
+    // MARK: - Properties
+
+    /// The color that is used to display the classification label in a user-interface
+    public let color: String?
```

Comment:
This may be the representation we want in the interface, but I just want to ask if returning a system color object is interesting.

Reply:
Or possibly providing an encoder/decoder. Because it may not be trivial to render this for users.

Reply:
Thanks for the suggestion. Makes sense to return a UIColor instead. Made a change to do that.